packages/ui/hooks/fuse/useExtraPoolInfo.ts

```diff
@@ -1,64 +1,55 @@
-import { useQuery } from 'react-query';
+import { useQuery } from '@tanstack/react-query';
 
 import { useMidas } from '@ui/context/MidasContext';
 
-export const useExtraPoolInfo = (comptrollerAddress: string) => {
+export const useExtraPoolInfo = (comptrollerAddress?: string) => {
   const { midasSdk, currentChain, address } = useMidas();
 
-  const { data } = useQuery(
-    ['ExtraPoolInfo', currentChain.id, comptrollerAddress],
+  return useQuery(
+    ['useExtraPoolInfo', currentChain.id, comptrollerAddress],
     async () => {
-      if (comptrollerAddress) {
-        const comptroller = midasSdk.createComptroller(comptrollerAddress);
-        const oracle = midasSdk.getPriceOracle(await comptroller.callStatic.oracle());
```

Comment:
This is what caused the error:
```
const oracle = midasSdk.getPriceOracle(await comptroller.callStatic.oracle());
```

If split up in two lines it works. Not sure why it just recently broke or why it does not work in the first place.

```
const oracleAddress = await comptroller.callStatic.oracle();
const oracle = midasSdk.getPriceOracle(oracleAddress);
```

As nothing relies on it later on, I just added it to the Promise Array in the line 35 in the changed file:

```
comptroller.callStatic
          .oracle()
          .then((oracleAddress) => midasSdk.getPriceOracle(oracleAddress)),
```

Reply:
Interesting. Nice deugging and overall refactoring. LG, apart from the single comment